### ISSUE TITLE
2.3.0 update broken

### ISSUE THREAD

**Issue by misulicus** [sudar/email-log#216]
@sudar  Just ran the update from wp.org an after that the website crashed:
```
[11-Apr-2019 11:27:35 UTC] PHP Fatal error:  Class 'Sudar\WPSystemInfo\SystemInfo' not found in D:\MAMP\htdocs\rcp-new\wp-content\plugins\email-log\include\Core\UI\Component\EmailLogSystemInfo.php on line 19
```

`email-log\vendor\sudar\wp-system-info\src` is empty

**Comment by misulicus** [comment]
i manually added the file from here to the plugin to get it back up: https://github.com/sudar/wp-system-info/blob/master/src/SystemInfo.php

**Comment by sudar** [comment]
Hello @misulicus 

Thanks for reporting the issue.

I just manually added that file to WordPress repo svn as well. This should prevent the error for people who are yet to download the plugin.

I am also going to be releasing v2.3.1 right away so that it fixes the error for people who have already updated.

**Comment by sudar** [comment]
Just released v2.3.1 as well.

Thanks for reporting the issue with full error message. It helped us to identify the issue as soon as possible.
